tests/TestRunner/upgrade_test.py

```diff
@@ -185,7 +191,7 @@ def download_old_binary(self, version, target_bin_name, remote_bin_name, makeExe
         os.rename(local_file_tmp, local_file)
         os.remove(local_sha256)
 
-        if makeExecutable:
+        if make_executable:
```

Comment:
Now we have a name clash between the parameter name and the function name. Need to rename one of them

Reply:
Thanks for the help! Changed it.